### ISSUE TITLE
Problem with the interactive charts

### ISSUE THREAD

**Issue by markziemann** [markziemann/mitch#9]
Data for axes are not correctly specified
![image](https://user-images.githubusercontent.com/5591840/82398274-d1453600-9a95-11ea-9a5e-fffc620f8374.png)

**Comment by markziemann** [comment]
This comes down to how x and y axes are specified in echarts4r which is odd to say the least

```
 p2 <- myres2 %>%
                tibble::rownames_to_column("model") %>%
                mutate(set = paste(set, setSize, p.adjustMANOVA , s.dist , sep = ",")) %>%
                e_charts(x=xx) %>%
                e_legend(show = FALSE) %>%
                e_x_axis(name = XCOL) %>%  # add x axis name
                e_y_axis(name = YCOL) %>%  # add y axis name
                e_scatter( serie= yy , bind = set , symbolSize = 10  ) %>%
                e_tooltip(formatter = htmlwidgets::JS("
                    function(params){
                    var vals = params.name.split(',')
                    return('<strong>' + vals[0] +
                    '</strong><br />s.x-axis: ' + parseFloat(params.value[0]).toFixed(2) +
                    '<br />s.y-axis: ' +  parseFloat(params.value[1]).toFixed(2)) +
                    '<br />setSize: ' + vals[1] +
                    '<br />p-adjust MANOVA: ' + Number(vals[2]).toPrecision(2) +
                    '<br />s.dist: ' + Number(vals[3]).toPrecision(2)
                }"))
```

**Comment by markziemann** [comment]
This has been resolved in v1.0.2
